externals/kyuubi-spark-sql-engine/pom.xml

```diff
@@ -98,6 +98,12 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.hive</groupId>
+            <artifactId>hive-service-rpc</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
we need it here, because it could provide org.apache.thrift.classes for the test.